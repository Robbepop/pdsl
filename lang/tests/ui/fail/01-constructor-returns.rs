#![feature(proc_macro_hygiene)]

use ink_lang as ink;

#[ink::contract(
    env = DefaultSrmlTypes,
    version = [0, 1, 0],
)]
mod noop {
    #[ink(storage)]
    struct Noop {}

    #[ink(impl)]
    impl Noop {
        #[ink(constructor)]
        fn new(&mut self) -> Self {}
    }
}

fn main() {}
