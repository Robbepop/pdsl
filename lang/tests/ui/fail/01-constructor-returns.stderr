error: constructors in ink! must have no specified return type
